db/schema.rb

```diff
@@ -14,7 +14,6 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
-  enable_extension "pg_stat_statements"
```

Comment:
Was this line remove by accident?

Reply:
Yes. This file was accidentally modified in [this commit](https://github.com/forem/forem/pull/14644/commits/7b6d6ff7cce892cfbf34166303210015a1796a73), I fixed it now.